traitsui/key_bindings.py

```diff
@@ -294,3 +260,48 @@ def _match_binding(self, binding, skip=frozenset()):
             for a_binding in self.bindings
             if a_binding not in skip and a_binding.match(binding)
         )
+
+
+class KeyBindingsHandler(ModelView):
+
+    bindings = List(Instance(KeyBinding))
+
+    def key_binding_for(self, binding, key_name):
+        """Returns the current binding for a specified key (if any)."""
+        if key_name != "":
+            for a_binding in self._match_binding(key_name, skip={binding}):
+                return a_binding
+
+        return None
+
+    def _match_binding(self, binding, skip=frozenset()):
+        """Return all KeyBinding instances that match the given binding.
+        """
+        return (
+            a_binding
+            for a_binding in self.bindings
+            if a_binding not in skip and a_binding.match(binding)
+        )
+
+    def _bindings_default(self):
+        bindings = list(set(self.model.root._get_bindings([])))
+        bindings.sort(key=lambda x: (x.binding1[-1:], x.binding1))
+        return bindings
+
+    traits_view = View(
+        [
+            Item(
+                "bindings",
+                style="readonly",
+                show_label=False,
+                editor=ListEditor(style="custom"),
+            ),
+            "|{Click on an entry field, then press the key to "
+            "assign. Double-click a field to clear it.}<>",
```

Comment:
double clicking to clear doesn't seem to work for me, but I may be doing something wrong

Reply:
Looks like it doesn on Wx, but was never hooked up in Qt.  I'll open an issue.

Reply:
See #1854.